File: android/PetSitterISI/app/src/main/java/com/example/petsitterisi/entitees/Service.java
package com.example.petsitterisi.entitees;

public class Service {

    private int id;
    private String description;
    private Double prixService;

    public Service(int id, String description, Double prixService) {
        this.id = id;
        this.description = description;
        this.prixService = prixService;
    }

    public int getId() {
        return id;
    }

    public String getDescription() {
        return description;
    }

    public Double getPrixService() {
        return prixService;
    }
}

File: backend/src/models/service.js
class Service {
    constructor(id, description, prix_service)
    {
        this._id = id
        this._description = description
        this._prix_service = prix_service
    }

    /* ######################################### GETTER #########################################*/

    get id(){
        return this._id
    }
    get description(){
        return this._description
    }
    get prix_service(){
        return this._prix_service
    }

    /* ######################################### SETTER #########################################*/

    set id(id){
        this._id = id
    }
    set description(description){
        this._description = description
    }
    set prix_service(prix_service){
        this._prix_service = prix_service
    }
}

module.exports = Service